### DISCUSSION TITLE
Global API for rename file?

### DISCUSSION THREAD

**Discussion by AlanBreck** [johnlindquist/kit#292] [Q&A] [answered]
Does Script Kit have a file rename function in the global API? Didn't see one listed, but wanted to check.

P.S. Script Kit is really great. At first I downloaded it, but had no ideas of how to use it in my workflow. Slowly but surely, though, I've started working it in.

**Comment by johnlindquist** [comment, marked as answer]
I'm pretty sure `mv` is what you're looking for.

Here's the `mv` API: https://github.com/shelljs/shelljs#mvoptions--source_array-dest

Here's a full script that uses it:
https://github.com/johnlindquist/kit/discussions/116

**Reply by AlanBreck** [reply]
Thanks! So are all shelljs commands automatically in scope?

**Reply by johnlindquist** [reply]
Most, if not all.

Here's a quick explanation of how to explore the api:

https://github.com/johnlindquist/kit/discussions/282#discussioncomment-755051
